testing_update_rpm.sh

```diff
@@ -0,0 +1,5 @@
+#! /bin/bash
+
+./scripts/build_rpm.sh
+yum -y remove rubygem-g5k-checks
```

Comment:
Why is this part needed?

Reply:
I used this script to build and install the gem on my test node.  I don't think it is necessary to bring into the master branch.